config/credhub_secret.go

```diff
@@ -77,3 +88,10 @@ func (r *credHubSecret) Remove(configServerInstanceName string, credHubPath stri
 
 	return nil
 }
+
+func validateCredHubPath(credHubPath string) error {
+	if strings.Count(credHubPath, "/") != 3 {
```

Comment:
This means anything along the lines of  `///`, `foo///`, `//foo/`... is valid. I'd be tempted to use a regex here and really lock this down.

Reply:
Good point! just updated it.